src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -93,12 +95,12 @@
 		<!-- LoanUse  -->
 		<kv-loading-paragraph
 			v-if="isLoading"
-			class="tw-mb-1.5" :style="{width: '100%', height: '5.5rem'}"
+			class="tw-mb-1.5 tw-flex-grow" :style="{width: '100%', height: '5.5rem'}"
 		/>
 
 		<loan-use
 			v-if="!isLoading"
-			class="tw-mb-2.5"
+			class="tw-mb-2.5 tw-flex-grow"
```

Comment:
This item is "growing" to take up extra available space with the result of pushing the following items to the bottom.